database/migrations/2020_03_15_152445_create_sales_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateSalesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('sales', function (Blueprint $table) {
```

Comment:
@okellogabrielinnocent Aren't we missing important fields for this table? Fields such as customer_id, staff_id?

Reply:
Was still working on it that's why I called it `WIP`(Work In Progress)

Reply:
Oh okay, I hadn't noticed the `WIP` label

Reply:
@jmatembu please take time and review